timescaledb/how-to-guides/hypertables/distributed-hypertables.md

```diff
@@ -291,15 +291,40 @@ the data over the data nodes, similar to traditional sharding.
 For more information about partitioning distributed hypertables, see the
 [About multi-node section][about-multinode].
 
+## Repartitioning distributed hypertables
+You can expand distributed hypertables by adding additional data nodes. To make
+use of the extra data nodes, you might need to repartition your distributed
+hypertable, to increase the number of space partitions. The new partitioning
+configuration only affects new chunks, which means the planner needs to work a
+bit differently to ensure that queries on your data nodes still produce the
+correct results. In this diagram, an extra data node was added during the third
+time interval, which means that the fourth time interval now includes four
+chunks instead of three:
```

Comment:
```suggestion
configuration only affects new chunks. In this diagram, an extra data node 
was added during the third time interval. The fourth time interval now includes 
four chunks, while the previous time intervals still include three:
```

I got a bit confused here and thought putting all the query planner information together might help